jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TimDepositController.java

```diff
@@ -105,6 +109,18 @@ public TimDepositController(OdeProperties odeProperties) {
       ? Boolean.parseBoolean(System.getenv("DATA_SIGNING_ENABLED_SDW"))
       : true;
 
+      // start the TIM ingest monitoring service if enabled
+      Boolean timIngestMonitoringEnabled = Boolean.valueOf(odeProperties.getProperty(ConfigEnvironmentVariables.ODE_TIM_INGEST_MONITORING_ENABLED));
+      if (timIngestMonitoringEnabled) {
+         logger.info("TIM ingest monitoring enabled.");
+         
+         ScheduledExecutorService scheduledExecutorService = Executors.newSingleThreadScheduledExecutor();
+         Long monitoringInterval = Long.valueOf(odeProperties.getProperty(ConfigEnvironmentVariables.ODE_TIM_INGEST_MONITORING_INTERVAL));
```

Comment:
If this property is not set, should we use a default value?

Reply:
That would make sense. I'm not sure what a sane default would be, though. My gut instinct is to set the monitor to run every hour by default (so 3600 here for the seconds in an hour), but that may be too frequent for our use case. What do you think would make a sane default, @dmccoystephenson?